src/Dashboard/App_Start/NinjectWebCommon.cs

```diff
@@ -1,86 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
-using System.Linq;
-using System.Web;
-using System.Web.Http;
-using System.Web.Mvc;
-using Dashboard.AppStart;
-using Dashboard.Infrastructure;
-using Microsoft.Web.Infrastructure.DynamicModuleHelper;
-using Ninject;
-using Ninject.Modules;
-using Ninject.Web.Common;
-
-[assembly: WebActivatorEx.PreApplicationStartMethod(typeof(NinjectWebCommon), "Start")]
-[assembly: WebActivatorEx.ApplicationShutdownMethodAttribute(typeof(NinjectWebCommon), "Stop")]
-
-namespace Dashboard.AppStart
-{
-    public static class NinjectWebCommon
-    {
-        private static readonly Bootstrapper Bootstrapper = new Bootstrapper();
-
-        public static IKernel Kernel 
-        { 
-            get 
-            { 
-                return Bootstrapper.Kernel; 
-            } 
-        }
-
-        /// <summary>
-        /// Starts the application
-        /// </summary>
-        [SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope")]
-        public static void Start()
-        {
-            // Registration must occur before application start. So use: WebActivator.PreApplicationStartMethod 
-            DynamicModuleUtility.RegisterModule(typeof(OnePerRequestHttpModule));
```

Comment:
Correct. That's part of why Autofac won over Ninject.  There's no intrinsic need for any of this.